lib/SourceMapDevToolPlugin.js

```diff
@@ -82,10 +82,15 @@ class SourceMapDevToolPlugin {
 		compiler.hooks.compilation.tap("SourceMapDevToolPlugin", (compilation) => {
 			new SourceMapDevToolModuleOptionsPlugin(options).apply(compilation);
 
-			compilation.hooks.afterOptimizeChunkAssets.tap("SourceMapDevToolPlugin", chunks => {
+			compilation.hooks.afterOptimizeChunkAssets.tap({
+				name: "SourceMapDevToolPlugin",
+				context: true
+			}, (context, chunks) => {
 				const moduleToSourceNameMapping = new Map();
 				const tasks = [];
+				const reportProgress = (context && context.reportProgress) ? context.reportProgress : () => {};
 
+				reportProgress(0.0, "SourceMapDevToolPlugin generation starting");
```

Comment:
Don't repeat the plugin name (SourceMapDevToolPlugin) in the message, it's prepended anyway.